### DISCUSSION TITLE
在应用商店安装的 Mongodb，如何修改配置文件 mongod.conf，没有找到

### DISCUSSION THREAD

**Discussion by ghost** [1Panel-dev/1Panel#5779] [Q&A]
想配置副本集，不知道配置文件在哪里，请问如何解决🙏

**Comment by wanghe-fit2cloud** [comment]
进入应用安装目录，查看一下 .data 持久化目录中是否存在上述配置文件：
<img width="674" alt="image" src="https://github.com/user-attachments/assets/f778fe79-2da9-40d2-b8d3-7a48c1bf0d68">

<img width="1294" alt="image" src="https://github.com/user-attachments/assets/a6e4960c-f5de-44a4-af7e-833de98adf60">

**Reply by zhaoyuqiqi** [reply]
这个data文件夹下面不存在上述配置文件
